File: r/raindrops/raindrops.R
raindrops <- function(number) {
    dplyr::case_when(
        number %% 105 == 0 ~ "PlingPlangPlong",
        number %% 15 == 0 ~ "PlingPlang",
        number %% 21 == 0 ~ "PlingPlong",
        number %% 35 == 0 ~ "PlangPlong",
        number %% 3 == 0 ~ "Pling",
        number %% 5 == 0 ~ "Plang",
        number %% 7 == 0 ~ "Plong",
        TRUE ~ as.character(number)
    )
}

File: python/raindrops/raindrops.py
def convert(n):
    result = ''
    if (n % 3) == 0:
        result = result + 'Pling'
    if (n % 5) == 0:
        result = result + 'Plang'
    if (n % 7) == 0:
        result = result + 'Plong'
    else:
        result = str(n)
    return result
